static/js/main.js

```diff
@@ -419,6 +421,15 @@ function convertUnixSecsForInput(unixSeconds) {
     return offsetDate.toISOString().slice(0, 19);
 }
 
+function convertCoordinatesForInput(coordinates) {
+    if (!coordinates) {
+        return undefined;
+    }
+
+    // Remove leading "+" from coordinates.
+    return coordinates.replace('+', '');
```

Comment:
Make this more precise (I recommend with a regex).
You comment says it removes a leading '+', but your code removes and '+'.